etc/adminhtml/system/adyen_cc.xml

```diff
@@ -77,17 +77,22 @@
                 <tooltip>Enable installments for each credit card type.</tooltip>
                 <config_path>payment/adyen_cc/enable_installments</config_path>
             </field>
-            <field id="installments" type="textarea" translate="label" sortOrder="220" showInDefault="1"
-                   showInWebsite="1" showInStore="1">
+            <field id="installments" translate="label" sortOrder="220" showInDefault="1" showInWebsite="1"
+                   showInStore="1">
                 <label>Installments</label>
                 <depends>
                     <field id="enable_installments">1</field>
                 </depends>
-                <validate>required-entry</validate>
-                <comment><![CDATA[
-                    Generate the desired installments configuration <a href="">here</a> and paste it on this field.
-                ]]></comment>
-                <config_path>payment/adyen_cc/installments_code</config_path>
+                <comment>Configure your installment for each credit card type: Insert the minimum amount required to 
+                make the configured installment available in the amount range column.
+                </comment>
+                <tooltip>
+                    Example: if the amount range is configured to 100 and the number of installments to 4x, the shopper
+                    will see the 4x option only if the payment total is higher or equal than 100.
+                </tooltip>
+                <frontend_model>Adyen\Payment\Block\Adminhtml\System\Config\Field\Installments</frontend_model>
+                <backend_model>Adyen\Payment\Model\Config\Backend\Installments</backend_model>
+                <config_path>payment/adyen_cc/installments</config_path>
```

Comment:
we are changing the config_path to installments instead of installments_code. Can't we be backwards compatible ?

Reply:
We had changed it to installments_code in this feature branch to enter the format required by the generic component manually. But then decided to implement a converter instead and restored the original config_path. So now it is backwards compatible as it is unchanged.